wgs/zk/README.md

```diff
@@ -25,3 +25,13 @@
 
 
 ## Questions
+
+
+- Do we want to support forced withdrawals or forced state transitions?
+- How do we serialize deposits into rollups?
+- Do we need Acks when transferring tokens between Celestia and the rollup?
+- Are Acks isomorphic to Option 4 of the spec proposal?
+- Do we need an on-chain light client of Celestia on the snark account if we want to support snark account <-> snark account bridging?
```

Comment:
[nit][question] since SNARK is an acronym, should it be capitalized?

```suggestion
- Do we need an on-chain light client of Celestia on the SNARK account if we want to support SNARK account <-> SNARK account bridging?
```